docs/content/specs/shared/_index.md

```diff
@@ -222,6 +222,30 @@ This section includes **shared, non-functional requirements (SNFR)** for Bicep a
 
 <br>
 
+#### ID: SNFR25 - Category: Composition - Resource Naming
+
+Module owners **MUST** set the default resource name prefix for child, extension, and interface resources to the associated abbreviation for the specific resource as documented in the following CAF article [Abbreviation examples for Azure resources](https://learn.microsoft.com/azure/cloud-adoption-framework/ready/azure-best-practices/resource-abbreviations), if specified and documented. This reduces the amount of input values a module consumer **MUST** provide by default when using the module.
```

Comment:
Hey @matt-FFFFFF, just found this little nugget which requires updates on the Bicep side. Also cross-checked (as I wanted to use your implementation for PE also for Bicep) and saw you're using `pe` while the linked article proposes `pep`. I usually also always used `pe`, but I wonder if that causes confusion if we point something else out here. Thoughts?

Reply:
Yup we must use what's in the doc as an abbreviation. I looked through both modules we've been working on and this is the only change I could see. 

Aligning to the doc keeps out of the way of an awkward conversation as to why we want to be different. 

Let's just align 👍

I'll happily put PR's in to fix

Reply:
Actually it's even written out as `pep` just 2 lines below